python/graphstorm/model/edge_decoder.py

```diff
@@ -1351,8 +1351,8 @@ def out_dims(self):
         return 1
 
 class LinkPredictContrastiveRotatEDecoder(LinkPredictRotatEDecoder):
-    """ Decoder for link prediction designed for contrastive loss
-        using the RotatE as the score function.
+    f""" Decoder for link prediction designed for contrastive loss
+    using the RotatE as the score function.
```

Comment:
```suggestion
    f""" Decoder for link prediction designed for contrastive loss
            using the RotatE as the score function.
```

Reply:
Aligning the 2nd line to the start of the first letter of the 1st line will resolve the different fonts issue.